card/mod/standard/set/all/rich_html/form.rb

```diff
@@ -140,30 +140,53 @@ def process_edit_fields fields
       field_nest field, opts
     end.join "\n"
   end
+  ###
 
-  def form_for_multi
-    instantiate_builder(form_for_multi_input_name, card, {})
+  # If you use subfield cards to render a form for a new card
+  # then the subfield cards should be created on the new card not the existing
+  # card that build the form
+
+
+  def form
+    @form ||= begin
+      @form_root = true unless parent&.form_root
+      instantiate_builder(form_prefix, card, {})
+    end
   end
 
-  def form_for_multi_input_name
-    voo&.live_options&.dig(:input_name) || "card#{subcard_input_names}"
+  def reset_form
+    @form = nil
+    form
   end
 
-  def subcard_input_names
-    return "" if !form_root_format || form_root_format == self
-    return @parent.subcard_input_names if @parent.card == card
-    "#{@parent.subcard_input_names}[subcards][#{name_in_form}]"
+  def form_prefix
+    case
+    when (voo_prefix = form_prefix_from_voo) then voo_prefix         # configured
+    when form_root? || !form_root || !parent then "card"             # simple form
+    when parent.card == card                 then parent.form_prefix # card nests itself
+    else                                          edit_in_form_prefix
+    end
   end
 
-  # If you use subfield cards to render a form for a new card
-  # then the subfield cards should be created on the new card not the existing
-  # card that build the form
-  def name_in_form
-    relative_names_in_form? ? card.relative_name : card.raw_name
+  def form_prefix_from_voo
+    voo&.live_options&.dig :input_name
```

Comment:
so...arguably this option should also be renamed to `form_prefix` (or should at least be consistent with other nomenclature).